internal/util/streamrpc/streamer.go

```diff
@@ -0,0 +1,139 @@
+package streamrpc
+
```

Comment:
did we test the behaviour of 
1) what will streamer happen on balance?
2) how will streamer handle timeout?